src/cmd/unit_generic.cpp

```diff
@@ -655,14 +655,6 @@ void Unit::DeactivateJumpDrive()
         jump.drive = -1;
 }
 
-float copysign( float x, float y )
```

Comment:
Check out where these come from (the header file), you'll have to include the proper header. I think it's `<math.h>` in this case. The include would go in the cpp.

Reply:
I believe `math.h` is already included. Line 7 of `unit_generic.cpp` includes `"unit_generic.h"`, which includes `"vegastrike.h"` on line 40. In turn, `"vegastrike.h"`, line 38, includes `"vs_math.h"`. And that file includes `<math.h>`, on line 27 and again on line 45.

Do your findings match what I'm seeing?

Reply:
How's this?